content/src/service/validations/Validations.ts

```diff
@@ -121,12 +122,38 @@ export class Validations {
   }
 
   /** Validate that uploaded and reported hashes are corrects */
-  static readonly CONTENT: Validation = async ({ deployment, externalCalls }) => {
+  static readonly CONTENT_V3: Validation = async ({ deployment, externalCalls }) => {
```

Comment:
We had to remove the check for profile files from v3 as we are not sure that all existing v3 deployments fulfills that condition